lab_5_scraper/scraper.py

```diff
@@ -197,6 +360,11 @@ def parse(self) -> Union[Article, bool, list]:
         Returns:
             Union[Article, bool, list]: Article instance
         """
+        response = make_request(self.full_url, self.config)
+        if response.status_code == 200:
```

Comment:
check what is `response.ok` for this case and why we can use it here. Answer me in this comment